modules/admin-ui/src/main/webapp/scripts/app.js

```diff
@@ -117,6 +117,7 @@ angular.module('adminNg', [
 .config(['chosenProvider', function (chosenProvider) {
   chosenProvider.setOption({
     'search_contains': true,
```

Comment:
@KatrinIhler This sets search_contains for all drop down boxes globally. That is why I have removed the many "data-search_contain" attributes: They are simply redundant.